guesser.rb

```diff
@@ -0,0 +1,31 @@
+play_again = true
```

Comment:
This is good, only thing I would recommend is to add some checks. What if the user enters something different other than` too low`, `too high`, `yes` or `no`. Then the program will have unexpected behavior. This was not directly required in the instructions but can be good practice.

Reply:
By having these checks you essentially makes your program "bullet proof" by not allowing the user to break your program.